app/controllers/observer_controller/search_controller.rb

```diff
@@ -25,12 +25,7 @@ def pattern_search # :nologin: :norobots:
       :name, :project, :species_list, :herbarium_record
       ctrlr = type
     when :google
-      if pattern.blank?
-        redirect_to(action: :list_rss_logs)
-      else
-        search = URI.escape("site:#{MO.domain} #{pattern}")
-        redirect_to("https://google.com/search?q=#{search}")
-      end
+      site_google_search(pattern)
```

Comment:
How is this related to updating the gems?

Reply:
Codeclimate had 2 flavors of complaints about this method:
- ABC/Too Many Lines/Perceived Complexity etc.  To fix those, I extracted `site_google_search`.
- Then, the follow-on complaint about URI.escape. See https://rubocop.readthedocs.io/en/latest/cops_lint/#linturiescapeunescape. I fixed that by using `URI.encode1